Dockerfile

```diff
@@ -14,11 +18,15 @@ WORKDIR /app
 COPY requirements.txt ./
 RUN pip install --no-cache-dir -r requirements.txt
 
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONUNBUFFERED 1
-
-# Copy the current directory contents into the container at /app
+# Copy the current directory contents into the container at /home
 COPY . /app
 
+# Create a user and switch to it (Optional for security)
+# RUN adduser --disabled-password --gecos "" myuser
```

Comment:
what is this and why is this commented out?

Reply:
This is in case you want to run the container as a specific user, and it is never meant to be used, just to give you an idea of a security measure you can take.